config/settings.yml

```diff
@@ -0,0 +1,9 @@
+---
+:cfme_cloud_services:
+  :manifest_configuration:
+    :scheme: https
+    :host: cert.cloud.redhat.com
+    :port:
+    :path: /api/topological-inventory/cfme/manifest/
```

Comment:
Minor but I'd rather not rely on the trailing slash here, if someone modifies the settings I bet you they're not going to realize they need a trailing slash which will cause [this](https://github.com/RedHatCloudForms/cfme-cloud_services/pull/18/files#diff-48eb1d1f5f56bd5cfc6f41e603af1186R23) to fail

Reply:
yes, removed and as @Fryguy suggested - we are using  `File.join(uri_config.path,` to avoid it

Reply:
:laughing: I hadn't refreshed the page so I totally missed that, :+1: thanks @lpichler